source/index.html.md

```diff
@@ -798,6 +871,58 @@ Only one 3-D Secure version can be used for a given authorization.
   </dd>
 </dl>
 
+##### Scheme reference to series
+
+If the previous-in-series authorization was made via this API, you should use
+`series[previous][id]` to reference the previous-in-series authorization.
+Otherwise, you must obtain explicit approval from Clearhaus to use these raw
+scheme values grouped in <code>series[previous][mastercard]</code> and
+<code>series[previous][visa]</code>.
+
+The Mastercard specific reference to the series contains the following parts.
+
+<dl class="dl-vertical">
+  <dt>
+    series[previous][mastercard][tid]
+    <span class="type">[A-z0-9]{3}[A-z0-9]{6}[0-9]{4}[ ]{2}</span>
```

Comment:
In other places we use `A-Za-z`.

Reply:
Nice catch!

I preferred the shorter one here because the regex is already pretty long 🙁

WDYT? Should we adjust the rest or this one? Or keep it inconsistent (me not like)?

Reply:
https://stackoverflow.com/a/4923397

```
> /[A-z]/ =~ ']'
=> 0
```